be/src/Project/Services/Client/Client.Api/Program.cs

```diff
@@ -67,6 +67,8 @@
 
 builder.Services.RegisterDatabaseDependencies(builder.Configuration);
 
+builder.Services.RegisterEventBus(builder.Configuration);
```

Comment:
**question:** why do we need this?

Reply:
Hi @dawiddworak88 ,

We need it because we previously used the endpoint created in the identity api, not to confuse it, I send it by queue to client.api